### PR TITLE
Tooltip update

### PR DESCRIPTION
**Pull request by bagrub**
updates for tooltip, nav-item, and app header render for templates

### CODE REVIEW

**Review by dgibson666 — COMMENTED** [review]
App header + stories updates look good. 

Tooltip issues as discussed (caret offset from the edge and right-bottom/left-bottom alignment).

File: packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss [hunk 1]
```diff
@@ -133,124 +133,136 @@ cbp-tooltip {
...
   --cbp-tooltip-placement-transform-y: 0;
   --cbp-tooltip-caret-rotation: 135deg;
-  --cbp-tooltip-caret-left: calc(100% - (var(--cbp-tooltip-caret-offset) * 3));
+  --cbp-tooltip-caret-left: calc(100% - (var(--cbp-tooltip-caret-size) + var(--cbp-tooltip-caret-space)));
   --cbp-tooltip-caret-top: calc(0% - var(--cbp-tooltip-caret-offset))
 }
 
-&[alignment='left-top'] div[role=tooltip]{
-  --cbp-tooltip-placement-top: calc(125% + var(--cbp-tooltip-placement-offset, 0px));
+&[alignment='left-top']{
```
Comment (dgibson666) [inline packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss:235]: This case is still not working right and still has magic numbers. `--cbp-tooltip-placement-top` should be at 100% (plus modifiers) and  `--cbp-tooltip-placement-transform-y` at 0%

**Review by dgibson666 — COMMENTED** [review]
Looks really good now. Just one case not working, noted in code.

File: packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss (hunk 1 above)
Comment (bagrub) [reply packages/web-components/src/components/cbp-tooltip/cbp-tooltip.scss:235]: updating, small note: looks better(aligns with top of control) for top @ 0% with the transform-y @ 0%

**Review by dgibson666 — APPROVED** [review]
looks good

**Pull request merged**